plugins/obs-qsv11/QSV_Encoder_Internal.cpp

```diff
@@ -948,7 +953,7 @@ mfxStatus QSV_Encoder_Internal::Reset(qsv_param_t *pParams,
 	mfxStatus sts = ClearData();
 	MSDK_CHECK_RESULT(sts, MFX_ERR_NONE, sts);
 
-	sts = Open(pParams, codec);
+	sts = Open(pParams, codec, 0);
```

Comment:
When is Reset called and does hard-coding the adapter to 0 here have any repercussions?